### DISCUSSION TITLE
Logout document

### DISCUSSION THREAD

**Discussion by christaikobo** [authelia/authelia#8474] [Documentation Feedback] [answered]
There used to be a document about logout:
https://github.com/authelia/authelia/commit/100d598a0edfef6980e2cbf2af554f4217d27497

And it's removed somehow, can someone please tell me is the information still relevant?

**Comment by Crowley723** [comment, marked as answer]
Yup, the information is still relevant. I believe the reason it was removed is because it already exists in the OpenAPI docs. Unfortunately we make very little mention to it on the Authelia website. If you have a running instance of Authelia, navigate to [https://auth.example.com/api](https://auth.example.com/api) and you can view and interact with the api and api schema.

**Reply by christaikobo** [reply]
Thank you!

**Comment by james-d-elliott** [comment]
It was removed in the docs revamp probably partially unintentionally. most of the information was not overly useful and as we move into specs like OpenID Connect 1.0 we hope users leverage the standardized session management tools once they're made available.
